inspectit-oce-demo/agentconfig/customers-service/config.yml

```diff
@@ -1,2 +1,27 @@
 # See https://github.com/inspectIT/inspectit-oce/blob/0.1.M1/inspectit-oce-core/src/main/resources/config/default.yml
-# for all the configuration options.
\ No newline at end of file
+# for all the configuration options.
+inspectit:
+  self-monitoring:
+    enabled: true
+#  instrumentation:
```

Comment:
Should these commented out lines stay in here? if yes maybe you should update the descriptions on them.